src/pages/oncall-me/oncall-me.ts

```diff
@@ -0,0 +1,76 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams, ActionSheetController, ToastController } from 'ionic-angular';
+import { IrisProvider, OncallUser} from '../../providers/iris/iris';
+import { OncallTeamPage } from '../oncall-team/oncall-team';
+import { IrisInfoProvider } from '../../providers/iris_info/iris_info';
+import { LogoutProvider } from '../../providers/logout/logout';
+
+@Component({
+  selector: 'page-oncall-me',
+  templateUrl: '../oncall-user/oncall-user.html',
+})
+export class OncallMePage {
```

Comment:
We can also probably get rid of the `oncall-me.ts` and set `mePageBool` in `ionViewWillEnter` based on whether the user matches `this.irisInfo.username`

Reply:
We could, although since the two  pages fetch data slightly differently (getting the user value from memory or from nav params) it might end up being less clean to combine them. Additionally the difference I think helps with navigation (knowing if you are in a sub page of oncall tab or in the me tab). But if you feel strongly that we should combine them I would be fine with that too.